app/models/registration.rb

```diff
@@ -36,6 +37,13 @@ def week
 
   private
 
+  def destroy_purchased_tickets
+    ticket_purchased = TicketPurchase.where(conference_id: conference_id)
+    unless ticket_purchased.empty?
+      ticket_purchased.destroy_all
```

Comment:
Erm this would destroy all TicketPurchase for the conference. For all users. For all tickets.